hack/svc/multi-stream.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import time
+import re
+import getopt
+import sys
+from subprocess import Popen
+
+
+class strims_ffmpeg:
+    def __init__(self, hostAddress="0.0.0.0") -> None:

```

Comment:
Once this is all de-dented, let's add two [argparse](https://docs.python.org/3/howto/argparse.html#introducing-optional-arguments) flags, `host-address` and `stream-key`